master/docs/manual/cfg-wwwhooks.rst

```diff
@@ -25,6 +25,28 @@ The :file:`post_build_request.py` script in :file:`master/contrib` allows for th
 Run :command:`post_build_request.py --help` for more information.
 The ``base`` dialect must be enabled for this to work.
 
+Mercurial hook
+++++++++++++++
+
+The Mercurial hook uses the base dialect:
+
+.. code-block:: python
+
+   c['www'] = dict(
+       …,
```

Comment:
you use here unicode characters, and pygments does not seem to be able to "grok" it.  please use plain ascii `...`